app/profile/page.tsx

```diff
@@ -170,10 +169,11 @@ const Some: React.FC = () => {
                                 );
                             }
                         }
-                    } else if (RSVP?.status === "REJECTED") {
+                    } else if (RSVP?.status === "REJECTED" || RSVP?.status === "WAITLISTED") {
```

Comment:
Waitlisted Message: same as Rejected but change the end to be "we will reach out to you if anything changes" instead of "reach out to us if you have questions"